<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPanel2</class>
 <widget class="QWidget" name="MyPanel2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Wing Width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leWidth"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Chord Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLength"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Leading Edge Radius:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRadius"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Angle of Attack:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAngle"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This panel creates a simple airfoil profile in a surrounding fluid based on the given parameters.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="cbMesh">
     <property name="toolTip">
      <string>Generate a mesh using boundary layers and hex meshing.</string>
     </property>
     <property name="text">
      <string>Generate Mesh</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leRadius</tabstop>
  <tabstop>leLength</tabstop>
  <tabstop>leWidth</tabstop>
  <tabstop>leAngle</tabstop>
  <tabstop>cbMesh</tabstop>
  <tabstop>pbApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
